cluster/addons/metrics-server/metrics-server-deployment.yaml

```diff
@@ -65,6 +65,19 @@ spec:
         - containerPort: 443
           name: https
           protocol: TCP
+        readinessProbe:
+          httpGet:
+            path: /
+            port: 443
+            scheme: HTTPS
+          periodSeconds: 10
```

Comment:
We should copy the readiness probe config from the upstream manifest: https://github.com/kubernetes-sigs/metrics-server/blob/release-0.4/manifests/base/deployment.yaml#L32-L38

Reply:
done